modulemd/v2/tests/test-modulemd-modulestream.c

```diff
@@ -66,6 +66,24 @@ module_stream_test_construct (ModuleStreamFixture *fixture,
       g_assert_null (modulemd_module_stream_get_stream_name (stream));
 
       g_clear_object (&stream);
+    
+    
+      /*Test that the new() function works with no module or stream*/
+    
+      stream = modulemd_module_stream_new (version, "foo", "latest");
```

Comment:
This doesn't match the comment you made above it. You wrote "Test that the new() function works with no module or stream" and then you created `stream` with the module name "foo" and stream name "latest".

This is just a copy of the test at line 31.

Reply:
Also, the test at line 58 handles the case of no module or stream, so I'm not sure what this patch is meant to accomplish.